content/posts/_index.md

```diff
@@ -11,17 +11,5 @@ bookCollapseSection: true
 - [March 2021](/posts/march-2021)
 - [February 2021](/posts/february-2021)
 - [January 2021](/posts/january-2021)
-- [December 2020](/posts/december-2020)
-- [November 2020](/posts/november-2020)
-- [October 2020](/posts/october-2020)
-- [Fedora ELN](/posts/fedora-eln)
```

Comment:
isn't this line removed by accident?

Reply:
Yes, thanks. Anyway, the text in the `content/posts/_index.md` seems to be ignored by the current theme we use (book) so it makes no sense to keep it there. I removed it in an additional commit.